README.md

```diff
@@ -1,149 +1,39 @@
-```
-sudo apt-get install cmake
-sudo add-apt-repository ppa:kalakris/cmake
-sudo apt-get update
-sudo apt-get upgrade
-```
+## Getting Started
+See the [Getting Started](https://smartdevicelink.com/en/guides/core/getting-started/) guide in the developer portal for instructions on how to build and start SDL Core.
 
-## Required RPCs
-There are several RPCs that are "required" to be implemented in order for SDL to work across vehicle manufacturers and applications, listed below.  The RPC specification can be found in the [Mobile API Spec](src/components/interfaces/MOBILE_API.xml).
+## Connect a Web HMI
+After setting up SDL Core, you will need to connect it to an external HMI. The HMI component of SDL needs to be developed separately from this project, and should be tailored to your specific head unit. Several sample browser-based HMIs are available for testing purposes:
 
-  * RegisterAppInterface
-  * UnregisterAppInterface
-  * SetGlobalProperties
-  * ResetGlobalProperties
-  * AddCommand
-  * DeleteCommand
-  * AddSubMenu
-  * DeleteSubMenu
-  * ShowAppMenu
-  * CreateInteractionChoiceSet
-  * PerformInteraction
-  * DeleteInteractionChoiceSet
-  * Alert
-  * Show
-  * SetMediaClockTimer
-  * SubscribeButton
-  * UnsubscribeButton
-  * ChangeRegistration
-  * GenericResponse
-  * SystemRequest
-  * OnHMIStatus
-  * OnAppInterfaceUnregistered
-  * OnButtonEvent
-  * OnButtonPress
-  * OnCommand
-  * OnDriverDistraction
-  * OnPermissionsChange
-  * OnLanguageChange
-  * OnSystemRequest
-  * Speak
+  1. The [Generic HMI](https://github.com/smartdevicelink/generic_hmi) (preferred) is a reference implementation for an SDL-connected HMI, including examples of many of the UI elements and templates available to SDL applications.
+  2. The [SDL HMI](https://github.com/smartdevicelink/sdl_hmi) emulates a full infotainment system which includes a simplified SDL component
 
 ## Frequently Asked Questions
 
-If you have any questions regarding the setup of SDL Core or regarding the integration of SDL into an infotainment system, first visit our [SDL Core FAQ](https://smartdevicelink.com/en/guides/core/faq/). If you still have questions, you can post a question in the `core-help` channel of our Slack group (sign up [here](http://slack.smartdevicelink.com/))
-
-## Test Coverage
-### Used technologies
-  * GCOV - test coverage program.
-  * LCOV - graphical front-end for GCC's coverage testing tool for gcov.
-  * codecov.io - service for assembling code coverage and representing it in a clear for reading form.
+If you have any questions regarding the setup of SDL Core or regarding the integration of SDL into a head unit, first visit our [SDL Core FAQ](https://smartdevicelink.com/en/guides/core/faq/). If you still have questions, you can post a question in the `core-help` channel of our Slack group (sign up [here](http://slack.smartdevicelink.com/)).
```

Comment:
Might be good to link to manticore for people who want to test an app and don't necessarily need to build core?